src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+function averageAge(people) {
+  return people.reduce(
+    (acc, curr) => acc + (curr.died - curr.born), 0
+  ) / people.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(
+    person => person.sex === 'm'
+  );
+  const menByCentury = men.filter(
+    person => Math.ceil(person.died / 100) === century
+  );
```

Comment:
No need to filter array twice and then select one of them, just select needed condition inside filter method